spec/request/v0/facilities/ccp_request_spec.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require File.expand_path(
+  Rails.root.join(
+    'spec',
+    'support',
+    'shared_contexts',
+    'facilities_ppms.rb'
+  )
```

Comment:
Surprises me you'd have to load it this way -- guessing its because we're not loading spec/support/* in the rspec helper, right?

Reply:
[This](https://github.com/department-of-veterans-affairs/vets-api/blob/4129-facility-locator-urgent-care-service-type-splits/spec/rails_helper.rb#L13-L23) and [This](https://github.com/department-of-veterans-affairs/vets-api/blob/4129-facility-locator-urgent-care-service-type-splits/spec/spec_helper.rb#L4-L15) could be replaced with 
```
Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
```